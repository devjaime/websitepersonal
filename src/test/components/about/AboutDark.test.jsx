import React from 'react';
import {render} from '@testing-library/react';
import About from '../../../components/about/AboutDark';

describe('Test component <AboutDark />', () =>{

    test(' print message "Im Jaime Hernandez"', ()=>{

        const message = "I'm Jaime Hernandez"
        const wrapper = render(<About />)
    }) 
} )