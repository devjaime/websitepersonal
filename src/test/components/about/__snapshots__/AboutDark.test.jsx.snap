// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component <AboutDark />  Test snapshot 1`] = `
<Fragment>
  <div
    className="about_inner"
  >
    <div
      className="left"
    >
      <img
        alt=""
        src="img/thumbs/1-1.png"
      />
      <div
        className="image"
        style={
          Object {
            "backgroundImage": "url(img/thumbs/1-1.png)",
          }
        }
      />
    </div>
    <div
      className="right wow fadeInLeft"
      data-wow-delay="0.3s"
      data-wow-duration="1.2s"
    >
      <div
        className="short"
      >
        <h3>
           I'm Jaime Hernandez
        </h3>
        <h5>
          Software Engineer 
          <span
            className="theme-color"
          >
            Full & Stack 
          </span>
          based in 
          <span
            className="theme-color"
          >
            Chile
          </span>
        </h5>
        <p>
          I create full stack web software, currently I specialize in python and go from the backend and data science side. Teamwork under different methodologies, scrum, kanban among others
        </p>
      </div>
      <div
        className="extra"
      >
        <h3
          className="title"
        >
          Personal Info
        </h3>
        <div
          className="list"
        >
          <ul>
            <li>
              <p>
                <span>
                  Name :
                </span>
                 Jaime Hernandez
              </p>
            </li>
            <li>
              <p>
                <span>
                  Address :
                </span>
                 Santiago of Chile
              </p>
            </li>
            <li>
              <p>
                <span>
                  Age :
                </span>
                 38 Years
              </p>
            </li>
            <li>
              <p>
                <span>
                  Phone :
                </span>
                 +56949288019
              </p>
            </li>
            <li>
              <p>
                <span>
                  Nationality :
                </span>
                 Chilean
              </p>
            </li>
            <li>
              <p>
                <span>
                  mail :
                </span>
                 hernandez.hs@gmail
              </p>
            </li>
            <li>
              <p>
                <span>
                  Freelance :
                </span>
                 Available
              </p>
            </li>
            <li>
              <p>
                <span>
                  Languages :
                </span>
                 Spanish, English
              </p>
            </li>
          </ul>
        </div>
        <div
          className="beny_tm_button color-outline"
        >
          <a
            download={true}
            href="https://drive.google.com/file/d/1eOrKXKZUkNYNhpB_q0HvMg22DXxWrXCK/view?usp=sharing"
          >
            <span
              className="wrapper"
            >
              <span
                className="first"
              >
                Download CV
              </span>
              <span
                className="second"
              >
                Download CV
              </span>
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <Skills />
</Fragment>
`;
